examples/live_stream/listener.c

```diff
@@ -198,8 +198,10 @@ int main(int argc, char *argv[ ])
 			fprintf(stderr,"frame sequence = %lld\n", frame_sequence++);
 			h1722 = (seventeen22_header *)((uint8_t*)frame + sizeof(eth_header));
 			length = ntohs(h1722->length) - sizeof(six1883_header);
-			write(1, (uint8_t *)((uint8_t*)frame + sizeof(eth_header) + sizeof(seventeen22_header) +
+			rc = write(1, (uint8_t *)((uint8_t*)frame + sizeof(eth_header) + sizeof(seventeen22_header) +
```

Comment:
Thanks for your comment @pinealservo. I have added a new commit to print a message when a non-success code is returned.

Regards,
Roland Hii